# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/api/FileSystemManager.h
# Commit: 214c4d6bf7f9
# Full Hash: 214c4d6bf7f98a332c4ab81d325d9245ae810955
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-10-20 21:51:26
# Regressor Bug: 1792245
# File Overlap Count: 2
# Description:
#   Bug 1792245 - Add ErrorResult to FileSystemRequestHandler methods; r=dom-storage-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D159572
# ==============================================================================

diff -r 330c69218a93 -r 214c4d6bf7f9 dom/fs/api/FileSystemManager.h
--- a/dom/fs/api/FileSystemManager.h	Thu Oct 20 07:33:41 2022 +0300
+++ b/dom/fs/api/FileSystemManager.h	Thu Oct 20 04:21:30 2022 +0000
@@ -8,6 +8,7 @@
 #define DOM_FS_CHILD_FILESYSTEMMANAGER_H_
 
 #include <functional>
+
 #include "mozilla/UniquePtr.h"
 #include "nsCOMPtr.h"
 #include "nsCycleCollectionParticipant.h"
@@ -54,7 +55,7 @@
       std::function<void(const RefPtr<FileSystemManagerChild>&)>&& aSuccess,
       std::function<void(nsresult)>&& aFailure);
 
-  already_AddRefed<Promise> GetDirectory(ErrorResult& aRv);
+  already_AddRefed<Promise> GetDirectory(ErrorResult& aError);
 
  private:
   virtual ~FileSystemManager();